CODING_STANDARDS.md

```diff
@@ -167,6 +167,14 @@ be part of the user-facing API. This typically applies to symbols used in templa
 Additionally, the `@docs-private` JsDoc annotation can be used to hide any symbol from the public
 API docs.
 
+
+#### Getters and Setters
+* Always use a `readonly` property instead of a getter (with no setter) when possible.
```

Comment:
Can you give an example? My brain is struggling to parse the use case

Reply:
Added